enhancements/installer/kubernete-slb-owned-api-loadbalancers.md

```diff
@@ -0,0 +1,508 @@
+---
+title: kubernetes-slb-owned-api-loadbalancers
+authors:
+  - "@abhinavdahiya"
+reviewers:
+  - "@sttts"
+  - "@smarterclayton"
+approvers:
+  - "@sttts"
+  - "@smarterclayton"
+creation-date: 2020-05-06
+last-updated: 2020-08-26
+status: implementable
+see-also:
+  -   
+replaces:
+  - 
+superseded-by:
+  - 
+---
+
+# Kubernetes Service Owned API Loadbalancers
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
```

Comment:
Can you also add a TODO item for discussing bare metal IPI, as well as the assisted installation [1] for bare metal?

[1] https://github.com/openshift/enhancements/blob/master/enhancements/installer/connected-assisted-installer.md